### PR TITLE
Create AUTHORS file

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Felix Thaler (fthaler), ETH Zurich (CSCS)
```
Comment (havogt) [inline AUTHORS:1]: Ordered by first contribution. Please fight, I don't remember...

Comment (havogt) [reply AUTHORS:1]: Or not if you don't care...

Comment (lukasm91) [reply AUTHORS:1]: I started, Felix took over when i was on holiday. But I don't care :)

Comment (havogt) [reply AUTHORS:1]: Ok, I'll swap.

Comment (havogt) [reply AUTHORS:1]: Anywhere it's nowhere written what the order means ;)

Comment (fthaler) [reply AUTHORS]: @havogt you have to add yourself, you are going to contribute the authors file ;P

**Review by fthaler — APPROVED** [review]

**Pull request merged**